themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -68,6 +68,28 @@
       <?php foreach (($top = $results->getRecommendations('top')) as $current): ?>
         <?=$this->recommend($current)?>
       <?php endforeach; ?>
+      <?php foreach (($side = $results->getRecommendations('side')) as $current):
```

Comment:
Minor nitpick: I see no reason to set `$side` here since the variable is never used.

Reply:
True, I was copying the pattern above, but $top might be used within the helper.  Removed.